### PR TITLE
태그 팔로우 gql

### PR DESCRIPTION
**Pull request by robin-maki**

### CODE REVIEW

**Comment by robin-maki** [comment]
Current dependencies on/for this PR:
* main
  * **PR #371** <a href="https://app.graphite.dev/github/pr/penxle/penxle/371?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/penxle/penxle/371?utm_source=stack-comment).

File: apps/penxle.com/prisma/schema.prisma [hunk 1]
```diff
@@ -408,11 +408,24 @@ model Tag {
   children  TagHierarchy[] @relation("children")
   userMutes UserTagMute[]
   posts     PostTag[]
+  follows   TagFollow[]
```
Comment (devunt) [inline apps/penxle.com/prisma/schema.prisma:411]: P2: `followers` ?

File: apps/penxle.com/src/lib/server/graphql/schemas/tag.ts [hunk 2]
```diff
@@ -97,6 +109,54 @@ builder.mutationFields((t) => ({
...
   }),
 
+  followTag: t.withAuth({ user: true }).prismaField({
+    type: 'Tag',
+    args: { input: t.arg({ type: FollowTagInput }) },
+    resolve: async (query, _, { input }, { db, ...context }) => {
+      const tag = await db.tag.findUniqueOrThrow({
+        ...query,
+        where: { id: input.tagId },
+      });
```
Comment (devunt) [inline apps/penxle.com/src/lib/server/graphql/schemas/tag.ts:119]: P2: 요거 제일 밑에서 바로 리턴하게 내려도 되는거 아닌가용

File: apps/penxle.com/src/lib/server/graphql/schemas/tag.ts [hunk 3]
```diff
@@ -97,6 +109,54 @@ builder.mutationFields((t) => ({
...
+  }),
+
+  unfollowTag: t.withAuth({ user: true }).prismaField({
+    type: 'Tag',
+    args: { input: t.arg({ type: UnfollowTagInput }) },
+    resolve: async (query, _, { input }, { db, ...context }) => {
+      const tag = await db.tag.findUniqueOrThrow({
+        ...query,
+        where: { id: input.tagId },
+      });
```
Comment (devunt) [inline apps/penxle.com/src/lib/server/graphql/schemas/tag.ts:147]: P2: 위와 마찬가지

**Review by devunt — CHANGES_REQUESTED** [review]

**Review by devunt — APPROVED** [review]

**Pull request merged**